### PR TITLE
Fix typos & formatting, clarify wording.

### PR DESCRIPTION
**Pull request by grumpasaurusrex**
- Fix typos
- Fix formatting, indentation, heading sizes
- Clarify wording of some sentences
- Order companies alphabetically

### CODE REVIEW

File: climate/strike.md [hunk 1]
```diff
@@ -9,62 +9,63 @@ permalink: /climate-strike/
...
-
-3. Post on social media your intent to walk out before September 20. On September 20, post photos of the walkout in your city  with the hashtag [#TechClimateStrike](https://twitter.com/search?q=%23TechClimateStrike&src=typed_query), preferably while wearing your company's branded clothing. If you're walking out with other tech workers, post a group photo (with everyone's consent, of course)
+2. Before September 20, talk to your colleagues, post to your company slack or message board, share on Twitter, and let people know why you're joining the Global Climate Strike. Use the hashtag [#TechClimateStrike](https://twitter.com/search?q=%23TechClimateStrike&src=typed_query).
 
-It is important that we build social media buzz so that we can get media attention and further amplify the youth's message.
+3. On September 20, post photos of the walkout in your city  with the hashtag [#TechClimateStrike](https://twitter.com/search?q=%23TechClimateStrike&src=typed_query), preferably while wearing your company's branded clothing. If you're walking out with other tech workers, post a group photo (with everyone's consent, of course).
```
Comment (kidmillions) [inline climate/strike.md]: edouble space

File: climate/strike.md [hunk 2]
```diff
@@ -9,62 +9,63 @@ permalink: /climate-strike/
...
+- Twitter
+
 
 ## Social media
 
-- Google: https://twitter.com/GoogleWAC
-- Microsoft: https://twitter.com/MsWorkers4
-- Amazon: https://twitter.com/AMZNforClimate
-- Tech Workers Coalition: https://twitter.com/techworkersco
+- Amazon: [https://twitter.com/AMZNforClimate](https://twitter.com/AMZNforClimate)
```
Comment (kidmillions) [inline climate/strike.md]: great, thanks for this.

Comment (shushugah) [reply climate/strike.md]: What is the difference?

**Comment by shushugah** [comment]
Small note, the `/climate/press-kit.md` reference was deleted, but the file itself was not deleted.

**Pull request merged**